packages/amplify-cli/src/extensions/amplify-helpers/get-amplify-version.ts

```diff
@@ -0,0 +1,7 @@
+import * as path from 'path';
+import { JSONUtilities, $TSAny } from 'amplify-cli-core';
+
+export const getAmplifyVersion = (): string => {
```

Comment:
There's going to be duplicate logic once this is merged: https://github.com/aws-amplify/amplify-cli/pull/7834/files#diff-a536d7e42a68805a2355f24a1186275d46f210bad20c4b5465ab3592ad277da4R63